apps/arena/lib/arena/entities.ex

```diff
@@ -55,7 +55,10 @@ defmodule Arena.Entities do
         visible_players: [],
         on_bush: false,
         bounties: [],
-        bounty_selected: false
+        bounty_selected: false,
+        selected_bounty: nil,
+        default_bounty: nil,
+        bounty_completed: false
```

Comment:
Do we need all these booleans? Maybe we could merge some of them:
1. Do we need selected_bounty and default_bounty separately? Can't the default take its place then the user overrides it (select)?
2. Do we need a boolean for bounty_selected? Doesn't the nil in selected bounty suffix that?

Reply:
1. I deleted the default bounty and now we'll do the random pick when the pick time runs out and the `selected_bounty ` is nil as you suggested
2. nope, this is related to the 1. point where we used this boolean to actually pick a default bounty, removed